docs/SRS/SRS.tex

```diff
@@ -338,6 +339,62 @@ \subsubsection{Authentication}
 	      \end{enumerate}
 \end{enumerate}
 
+\subsubsection{Services}
+\begin{enumerate}[resume*=business_events]
+	\item The user wants to add available auto shop services to the shop profile
+	      \begin{enumerate}[VP\arabic*.]
+		      \item Viewpoint: Vehicle Owner
+		            \begin{enumerate}
+			            \item[] N/A
+		            \end{enumerate}
+		      \item Viewpoint: Auto Repair Shop Owner
+		            \begin{enumerate}
+			            \item The system shall allow the user to enter the name of the service
+			            \item The system shall allow the user to enter a description for the service
+			            \item The system shall allow the user to enter the estimated time for the service
+			            \item The system shall allow the user to enter the parts used for the service including quantity,
+			                  condition, build, and cost per part
+			            \item The system shall allow the user to enter the total price for the service
```

Comment:
I'm guessing some fields are optional here... do we want to add "optionally enter" or something similar for optional service fields

Reply:
I thought it was just showing that we are letting the user enter it.. doesn't mean they HAVE to